ci/test.sh

```diff
@@ -53,6 +53,10 @@ echoinfo "Test complete"
 
 echoinfo "Uploading test results"
 
+# Determine CODACY_REPORTER_VERSION once to avoid hitting the "get latest" API too frequently
```

Comment:
Doesn't `get.sh` cache everything after the first call?

Reply:
It caches the actual reporter code, but it hits the "get latest version" URL on every invocation. I suspect it causes API throttling of something to that effect.